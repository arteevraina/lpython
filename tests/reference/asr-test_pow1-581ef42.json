{
    "basename": "asr-test_pow1-581ef42",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pow1.py",
    "infile_hash": "a7ce7888e7e99bc753e16822749aaae4ee63641675990b3e2a34de10",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_pow1-581ef42.stderr",
    "stderr_hash": "32fe0eecb3687de0e976228815c25c96f3dc3e8f6491adb21c105ee6",
    "returncode": 2
}