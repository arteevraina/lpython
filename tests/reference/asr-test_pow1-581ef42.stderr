warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/errors/test_pow1.py:4:11
  |
4 |     print(pow(x,3))
  |           ^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
